mk/dependencies/yamllint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+if [ -x "$(command -v yamllint)" ]; then
+  echo "Yamllint is already installed"
+  exit
+fi
+  echo "Installing Yamllint"
+  set -x
+  sudo apt update --fix-missing
+  sudo env DEBIAN_FRONTEND=noninteractive apt install -y yamllint
+  echo "Yamllint has been installed"
```

Comment:
This won't work on macOS and other linux distributions, please, could you implement this script to be more OS agnostic?

I would also like to have pinned version of it.